### PR TITLE
Added exercise structure to README

### LINKED ISSUE #2: Consider consistency for the exercises
- Is there a style guide for Julia?
- Are these styles encouraged or enforced?
- Are there any conventions that we should adopt on this track for the sake of consistency?
- Can we enforce these?
- Is there a linter? Are there many? Should we use one?
- Is there a common convention for filenames? If not, what should our convention be?

Note that this is about the exercises (the test suites and code examples), not people's solutions.

### PR DESCRIPTION
**Pull request by SaschaMann**
As discussed in issue #2.

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,18 @@ _Document how to contribute to the Julia track._
...
+A pool of exercises can be found in the [x-Common repo](https://github.com/exercism/x-common).
+
+Exercises for the Julia track go in the `exercises` directory and should follow the following filename conventions: 
+
+`exercises/<name>/<name>.jl` Skeleton for the function that is called by the test suite. Provide (abstract) parameter and return types to ensure compatibility with the test suite.
+
+`exercises/<name>/runtests.jl` Test suite for the exercise. Group related tests using [testsets](http://docs.julialang.org/en/release-0.5/stdlib/test/#working-with-test-sets).
+
+`exercises/<name>/example.jl` Example solution for the exercise. It should follow the [Julia Style Guide](http://docs.julialang.org/en/release-0.5/manual/style-guide/).
+<!-- See Issue 2 for discussion -->
```
Comment (kytrinyx) [inline README.md]: Let's make this an actual link to the issue: https://github.com/exercism/xjulia/issues/2

File: README.md [hunk 2]
```diff
@@ -9,3 +9,18 @@ _Document how to contribute to the Julia track._
...
+
+(`<name>` refers to the exercise name)
```
Comment (kytrinyx) [inline README.md]: We've been using the word `slug` for this elsewhere. It's the URL-friendly identifier for a given exercise. The name of the exercise is what would go in the title of the README for it. So:

- **name**: Hello World
- **slug**: `hello-world`

We've actually struggled quite a lot with terminology, and thrashed around a lot so you'll probably see remnants of old names strewn around various codebases and issues and pull requests.

**Comment by SaschaMann** [comment]
Applied your suggestions. Not quite sure if that's what you meant with the slug-comment. Let me know if it's not.

**Comment by kytrinyx** [comment]
Yepp, this looks great! Thanks.

**Pull request merged**